### DISCUSSION TITLE
Previous versions of www/falkon-qtonly

### DISCUSSION THREAD

**Discussion by grahamperrin** [helloSystem/hello#506] [Show and tell]
Spun off from <https://github.com/helloSystem/hello/issues/161#issuecomment-1596238688>

<http://pkg.freebsd.org/FreeBSD:13:amd64/release_2/All/falkon-qtonly-22.12.0.pkg>

The downloaded package: 

* installs and runs on FreeBSD 13.2-RELEASE with packages from _latest_ 
* should be treated as unsupported in this context (release_2 is behind both latest and quarterly).

**Comment by probonopd** [comment]
Thanks @grahamperrin.

The way I understand it, 13.2 is minor version 2, and the 2 in "release_2" means that it is a package that was made when 13.2 came out.

In order to solve the "packages can disappear from one day to the next" issue, we might pin packages to release_X corresponding to the FreeBSD minor version for all packages, BUT...

* Graphics drivers get updated only once the previous version is no longer supported, so release_X packages won't neccessarily work with FreeBSD minor version X(!)
* Users would get no more (security) updates for packages

This is why I am hesitant to use release_X packages for the whole system.

It might be possible to fall back to release_X packages for missing packages in quarterly, but I want to avoid the complexity of per-package workarounds.
